docs/user-guide/deployments/deploy-on-kubernetes/monitoring/cluster-monitoring-deployment.md

```diff
@@ -0,0 +1,218 @@
+---
+keywords: [Kubernetes deployment, cluster, monitoring]
+description: Guide to deploying monitoring for GreptimeDB clusters on Kubernetes, including steps for self-monitoring and Prometheus monitoring.
+---
+
+# Cluster Monitoring Deployment
+
+After deploying a GreptimeDB cluster using GreptimeDB Operator, by default its components (Metasrv / Datanode / Frontend) expose a `/metrics` endpoint on their HTTP port (default `4000`) for Prometheus metrics.
+
+We provide two approaches to monitor the GreptimeDB cluster:
+
+1. **Enable GreptimeDB Self Monitoring**: GreptimeDB Operator will launch an additional GreptimeDB Standalone instance to collect metrics and logs from the GreptimeDB cluster.
```

Comment:
Here is the vector collect metrics and logs ？

Reply:
yeap